app/panel/components/Panel.jsx

```diff
@@ -217,7 +217,7 @@ class Panel extends React.Component {
 	_premiumSubscriber = () => {
 		const { loggedIn, user } = this.props;
 
-		return loggedIn && (user && user.subscriptionsPremium);
+		return loggedIn && (user && user.premiumAccess);
 	}
 
 	/**
```

Comment:
Can we rename this and the  matching `_plusSubscriber` helper to reflect that they now return access status rather than subscription status?

Reply:
Changed to `_hasPlusAccess()` and `_hasPremiumAccess()`!